### ISSUE TITLE
Overlapping player equipment

### ISSUE THREAD

**Issue by Skerxan** [DerpyProjectGroup/topaz#780]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: everything

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

For example, `Behemoth Suit` restricts usage of the leg slot and ofcourse its body slot. But due it covering the whole body bar head, it makes for double rendered body parts.

Also experiencing on:
```
Chocobo suit
Goblin suit
Mandragora suit
```

Basically any equipment with "suit" in its name.

Better explained on screenshots:
![00c4e14545](https://user-images.githubusercontent.com/33092280/137761094-f5d9ce19-2876-4020-96eb-2a501c4c2e26.png)
![d565edd771](https://user-images.githubusercontent.com/33092280/137761132-3d37112f-174a-4f31-9aba-2fb201dc6ff8.png)
![1ba2d42e2d](https://user-images.githubusercontent.com/33092280/137761158-87c5ba1e-ed32-493b-8203-dd4999cf09eb.png)
![4768ffd0d9](https://user-images.githubusercontent.com/33092280/137761178-3073ffe8-2317-4f7b-9e84-5f89f5a96d0a.png)

**Comment by Omnione** [comment]
This seems to be fixed now, tested October 12th 2025
Tested with all races and sizes
Pictures provided tested same race and face for reference.
<img width="354" height="359" alt="Image" src="https://github.com/user-attachments/assets/ace5b52a-2cb9-4553-a86e-118819b05b4d" />

<img width="363" height="355" alt="Image" src="https://github.com/user-attachments/assets/e13ed7fe-4245-4eba-8576-fcc878872a69" />

<img width="289" height="333" alt="Image" src="https://github.com/user-attachments/assets/1a191bd1-cb38-457a-8601-3ee775d3d5a5" />

<img width="419" height="372" alt="Image" src="https://github.com/user-attachments/assets/f20ee0ba-dd16-4deb-9fa4-c18350dac207" />
